api/gateway-operator/gateway/conditions.go

```diff
@@ -0,0 +1,33 @@
+package gateway
+
+import "github.com/kong/kubernetes-configuration/api/common/consts"
+
+// -----------------------------------------------------------------------------
+// Gateway - Status Condition Types
+// -----------------------------------------------------------------------------
+
+const (
+	// GatewayServiceType the Gateway service condition type
+	GatewayServiceType consts.ConditionType = "GatewayService"
```

Comment:
The `GatewayService` does not look like a condition type (`GatewayServiceReady` looks better.). However, we have been using it and changing it may be a breaking change.

Reply:
Yes. So I want to keep it this way for now.